pkg/instaclustr/config.go

```diff
@@ -52,6 +52,8 @@ const (
 	AWSEncryptionKeyEndpoint               = "/cluster-management/v2/resources/providers/aws/encryption-keys/v2/"
 	ListAppsVersionsEndpoint               = "%s/cluster-management/v2/data-sources/applications/%s/versions/v2/"
 	ClusterSettingsEndpoint                = "%s/cluster-management/v2/operations/clusters/v2/%s/change-settings/v2"
+	OpenSearchEgressRulesEndpoint          = "/cluster-management/v2/resources/applications/opensearch/egress-rules/v2"
+	OpenSearchEgressRuleDeleteEndpoint = "/cluster-management/v2/resources/applications/opensearch/egress-rules/v2/"
```

Comment:
Both constants are identical. Please delete the second one.

Reply:
fixed